[ERROR] Unknown Command: _cmd_build

Hi to all.
I'm trying to build my "first" python script with atoupdate feature.
I always get an error:

`(env) PS C:\Users\User\Documents\test35> pyupdater build --app-version=1.0.0 app.py
917 INFO: PyUpdater 2.3.2
[INFO] PyUpdater 2.3.2
924 DEBUG: Config Dir: C:\Users\User\Documents\test35\.pyupdater
924 DEBUG: Config DB: C:\Users\User\Documents\test35\.pyupdater\config.pyu
928 DEBUG: Building from python source file: app.py
928 DEBUG: Source file abs path: C:\Users\User\Documents\test35\app.py
929 DEBUG: Creating directory: C:\Users\User\Documents\test35\.pyupdater\work
949 DEBUG: App Info: {'type': 'script', 'name': 'C:\\Users\\User\\Documents\\test35\\app.py'}
950 DEBUG: Hook directory: c:\users\user\documents\test35\env\lib\site-packages\pyupdater\hooks
951 DEBUG: Make spec cmd: -F --name=win --specpath=C:\Users\User\Documents\test35\.pyupdater\spec --additional-hooks-dir
=c:\users\user\documents\test35\env\lib\site-packages\pyupdater\hooks C:\Users\User\Documents\test35\app.py
**953 ERROR: Unknown Command: _cmd_build
[ERROR] Unknown Command: _cmd_build**`

as you can see I'm working in a venv...
Is there something I'm missing?
Thanks, for support!
Gianluca

--- a/pyupdater/cli/__init__.py
+++ b/pyupdater/cli/__init__.py
@@ -90,13 +90,13 @@ def _real_main(args, namespace_test_helper=None):  # pragma: no cover
 def dispatch_command(args, pyi_args=None, test=False):
     # Turns collect-debug-info into collect_debug_info
     cmd_str = "_cmd_" + args.command.replace('-', '_')
-    try:
+    if hasattr(commands, cmd_str):
         cmd = getattr(commands, cmd_str)
         # We are just making sure we can load the function
         if test:
             return True
         cmd(args, pyi_args)
-    except AttributeError:
+    else:
         # This should only get hit by misconfigured tests.
         # "Should" being the key word here :)
         log.error('Unknown Command: %s', cmd_str)

--- a/pyupdater/pyinstaller_compat.py
+++ b/pyupdater/pyinstaller_compat.py
@@ -88,7 +88,8 @@ def pyi_makespec(pyi_args):  # pragma: no cover
         # using noqa below so landscape.io will ignore it
         _pyi_makespec.__add_options(parser)  # noqa
         _pyi_log.__add_options(parser)  # noqa
-        _pyi_compat.__add_obsolete_options(parser)  # noqa
+        if hasattr(_pyi_compat, '__add_obsolete_options'):
+            _pyi_compat.__add_obsolete_options(parser)  # noqa
         # End hacking
         opts, args = parser.parse_args(pyi_args)
         # We are hacking into pyinstaller here & are aware of the risks
@@ -103,7 +104,8 @@ def pyi_makespec(pyi_args):  # pragma: no cover
         # using noqa below so landscape.io will ignore it
         _pyi_makespec.__add_options(parser)  # noqa
         _pyi_log.__add_options(parser)  # noqa
-        _pyi_compat.__add_obsolete_options(parser)  # noqa
+        if hasattr(_pyi_compat, '__add_obsolete_options'):
+            _pyi_compat.__add_obsolete_options(parser)  # noqa
         # End hacking
         parser.add_argument('scriptname', nargs='+')
 
@@ -112,7 +114,8 @@ def pyi_makespec(pyi_args):  # pragma: no cover
         # We call init because it loads logger into the global
         # namespace of the Pyinstaller.log module. logger is used
         # in the Pyinstaller.log.__process_options call
-        _pyi_log.init()
+        if hasattr(_pyi_log, 'init'):
+            _pyi_log.init()
         # We are hacking into pyinstaller here & are aware of the risks
         # using noqa below so landscape.io will ignore it
         _pyi_log.__process_options(parser, args)  # noqa

